### DISCUSSION TITLE
WalletConnect is working for android android but not working for ios.

### DISCUSSION THREAD

**Discussion by rabinvg** [WalletConnect/walletconnect-monorepo#3545] [Developer Support]
Websocket fails to connect in ios for a project that is working in the android.

**Comment by mshahjahan-mufin** [comment]
Check if you have performed the necessary changes in project and plist files.

**Reply by MuckT** [reply]
I am seeing this with DApps in mobile browsers on iOS and not specifically in a swift or react native app.

**Reply by rabinvg** [reply]
@ShahjahanMirza92 I have tested it on the wallect connect example project. One test project that i created few weeks ago is working, but the real project id is not working but its working on the android. Strange.
